hamilflow/models/brownian_motion.py

```diff
@@ -181,7 +179,7 @@ def _trajectory(self, n_new_steps: int, seed: int) -> "npt.NDArray[np.float64]":
         )
 
         step_history = np.concatenate(
-            (np.expand_dims(self.initial_condition.x0, axis=0), step_history)
+            (np.expand_dims(self.initial_condition.x0, axis=0), step_history),
```

Comment:
Ruff adds comma to many function calls and instantiations even for single args. This is a bit strange. Is there a PEP for this style?